src/Compilers/CSharp/Portable/Symbols/LocalDeclarationKind.cs

```diff
@@ -16,7 +16,7 @@ internal enum LocalDeclarationKind : byte
         None,
 
         /// <summary>
-        /// User defined local variable declared by <see cref="LocalDeclarationStatementSyntax"/>.
+        /// User defined local variable declared by <see cref="LocalDeclarationStatementSyntax"/> or by "out var" argument.
         /// </summary>
```

Comment:
I will do this once and if I am confident we need it. We are not at this point yet. 

---
In reply to: [63748250](https://github.com/dotnet/roslyn/pull/11393#discussion_r63748250) [](ancestors = 63748250)